packages/api-explorer/src/components/Header/Header.tsx

```diff
@@ -50,6 +51,15 @@ interface HeaderProps {
 
 export const HEADER_REM = 4
 
+/**
+ * Renders the API Explorer header
+ * @param className
```

Comment:
Not sure I follow. The `HeaderProps` interface defines all the parameters that `Header` takes. Do you mean I should document them at the interface level instead of using JS Doc?

Reply:
We spoke about this offline. Properties should be documented at the interface level. The comment above the component should only describe what the component does. @jkaster @noahamac